vms/platformvm/camino_helpers_test.go

```diff
@@ -229,3 +249,30 @@ func generateKeyAndOwner(t *testing.T) (*secp256k1.PrivateKey, ids.ShortID, secp
 		Addrs:     []ids.ShortID{addr},
 	}
 }
+
+func stopService(t *testing.T, service *CaminoService) {
+	service.vm.ctx.Lock.Lock()
+	require.NoError(t, service.vm.Shutdown(context.TODO()))
+	service.vm.ctx.Lock.Unlock()
+}
+
+func generateTestUTXO(txID ids.ID, assetID ids.ID, amount uint64, outputOwners secp256k1fx.OutputOwners, depositTxID, bondTxID ids.ID) *avax.UTXO {
```

Comment:
`generateTestUTXO` is moved here from vms/platformvm/camino_service_test.go, because its just like other test-helper funcs. This func is also used outside of `camino_service_test.go`